src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+
+  if (days > 2 && days < 7) {
+    return days * costPerDay - 20;
+  }
+
+  if (days > 6) {
+    return days * costPerDay - 50;
+  }
+
+  if (days < 3) {
+    return days * costPerDay;
+  }
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 50, 20, 7, 2 and 3 are